src/helpers.c

```diff
@@ -1199,3 +1200,17 @@ char *get_tracking_dir(void)
 {
 	return sys_path_join(globals.state_dir, "bundles");
 }
+
+bool confirm_action(const char *warning_msg, const char *action)
+{
+	int response;
+
+	if (warning_msg) {
+		warn("%s\n", warning_msg);
+	}
+	info("Do you want to continue%s? (y/N): ", action ? action : "");
```

Comment:
I would rephrase this in a way that you don't need to have an argument for action.
For example:
```
The 3rd-party bundle you are about to install contains files with dangerous permission
Do you want to continue? (y/N): 
```

Reply:
done